### ISSUE TITLE
vue/valid-v-else false positive with nested element and text node

### ISSUE THREAD

**Issue by dcecile** [rashfael/eslint-plugin-vue-pug#21]
I have a `v-if` with an element and a text node inside, and it's causing a false positive for `vue/valid-v-else`.

Here's my minimal repro:

```vue
<template lang="pug">
div
  section(v-if="true")
    div
      div
    | text
  section(v-else)
    div Checking dependencies...
  footer
</template>

<script lang="ts">
export default {}
</script>
```

Here are the lint errors:

```
  7:11  error  'v-else' directives require being preceded by the element which has a 'v-if' or 'v-else-if' directive  vue/valid-v-else
  9:3   error  The template root requires exactly one element                                                         vue/no-multiple-template-root
```

I tried using the `eslint-plugin-vue` demo and wasn't able to reproduce there, so I'm guessing the issue is from this plugin?

---

Variations:

```vue
<template lang="pug">
div
  div
    div
      div
    | text
  div(v-if="true")
  div(v-else)
</template>

<script lang="ts">
export default {}
</script>
```

```
  8:3  error  The template root requires exactly one element  vue/no-multiple-template-root
```

```vue
<template lang="pug">
div
  page-header
    template
      div
        div
      | TEXT

    template(#custom)
</template>

<script lang="ts">
export default {}
</script>
```

```
  9:14  error  'v-slot' directive must be owned by a custom element, but 'div' is not  vue/valid-v-slot
```

**Comment by rashfael** [comment]
Pipes after an outdent closing parent elements early has been fixed in 0.6.2.
The patch also fixes this issue.

**Comment by dcecile** [comment]
Thanks @rashfael! Finally upgraded & worked perfectly
